app/services/etherscan_service.rb

```diff
@@ -13,10 +13,30 @@ def is_proof_member?(address)
       address: address
     )
 
-    if response.dig('result')&.last&.dig('to')&.downcase == address.downcase
+    transactions = response.dig('result')
+
+    inCount = 0
+    outCount = 0
```

Comment:
Although these names will work, in ruby variable naming convention is actually to. use underscores between words... 

so `in_count`, `out_count`, `to_address`, `from_address` ect